### DISCUSSION TITLE
Firefox emits several warnings regarding WebGL and user-select

### DISCUSSION THREAD

**Discussion by karussell** [maplibre/maplibre-gl-js#2752] [General]
Besides a few other warnings I get:

> WebGL warning: texImage: Alpha-premult and y-flip are deprecated for non-DOM-Element uploads.

and more often I do get:

> Error in parsing value for ‘user-select’.  Declaration dropped.

Identifying the reason for these warnings does not seem possible as clicking on them does not point to some `user-select` css.

It can be reproduced [in the official examples](https://maplibre.org/maplibre-gl-js-docs/example/zoomto-linestring/) already when loading the page. And one can produce `Error in parsing value for ‘user-select’.  Declaration dropped.` three times when clicking on the "Zoom to bounds" button.

How can I avoid this?

**Comment by HarelM** [comment]
This is described here I think:
https://github.com/maplibre/maplibre-gl-js/issues/2030
There's an open PR to solve this, I need to understand what's still waiting in order to get it merged.
https://github.com/maplibre/maplibre-gl-js/pull/2297
